server/server/views/profile-history.py

```diff
@@ -0,0 +1,41 @@
+from cornice.resource import Service
```

Comment:
This file name does not meet the PEP8 ( I talk about profile-history.py) [PEP8](https://www.python.org/dev/peps/pep-0008/#package-and-module-names)